app/routes/AppContainer.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from "react"
+import PropTypes from "prop-types"
+import { connect } from "react-redux"
+import { ipcRenderer } from "electron"
+
+import {assignmentFetchInfo} from "../modules/assignment/actions/assignment-fetch-info"
+import {setAssignmentURL} from "../modules/assignment/actions/assignment-set-url"
+import {settingsUpdateUserState} from "../modules/settings/actions/settings-update-user-state"
+
+class AppContainer extends Component {
+  constructor (props) {
+    super(props)
+
+    this.props.updateUserState() // Get user if already logged in from previous session
+
+    ipcRenderer.on("open-url", (event, assignmentURL) => {
```

Comment:
Wrapper around all components so we can listen for ipc events on any page. Gets triggered if deep link is open, and sends initialized message once component is mounted